python/ray/serve/tests/test_api.py

```diff
@@ -364,40 +374,31 @@ class A:
         def __del__(self):
             signal.send.remote()
 
-    A.deploy()
-    A.delete()
+    serve.run(A.bind(), name="A")
+    serve.delete("A")
```

Comment:
Nitpick (non-blocker): can probably refactor the app name "A" and share it between the two calls 🙃